io.botfather.Botfather.yaml

```diff
@@ -0,0 +1,57 @@
+app-id: io.botfather.Botfather
+branch: stable
+runtime: org.kde.Platform
+runtime-version: '5.11'
+sdk: org.kde.Sdk
+command: botfather
+tags:
+  - proprietary
+finish-args:
+  - "--filesystem=home:rw" # let user use their scripts properly
+  - "--require-version=0.11.4" # fixes kde file saving
+  - "--socket=x11" # show windows using X11
+  - "--share=ipc" # share IPC namespace with the host (necessary for X11)
+  - "--device=dri" # OpenGL rendering (for CEF)
+  - "--socket=pulseaudio" # play sound with PulseAudio
+  - "--share=network" # access the network (CEF)
+  - "--device=all" # for /dev/uinput (mouse and keyboard input)
+  - "--talk-name=org.freedesktop.ScreenSaver"
+build-options:
+  env:
+    CEF_ROOT: "/app/CEF"
+    LD_LIBRARY_PATH: "/app/lib"
```

Comment:
This *shouldn't* be needed. If so I think its a runtime bug.

Reply:
They are indeed not needed. They are leftovers from the manifest I use to build my (prebuilt) binaries.